### ISSUE TITLE
Menu - Hover effect Stretch

### ISSUE THREAD

**Issue by abrarpathan19** [magento/magento2#19652]
### Preconditions (*)

1.Magento 2.3 

### Steps to reproduce (*)

Currently, when we go on the right of the "Tops"(second level category menu) link in the menu, if you go directly on the right (without passing on the letter “Tops”) it is not displaying the sub-menu.
1. Hover on Women (Top menu main category)
2. Go directly on the right of the "Tops" button (without hovering over the Tops text) and try the hover, it won't show the sub-menu.
(It only show the sub-menu "Tops" if you hover over the Tops text in the button.)
3. http://prntscr.com/lsb7sw

In short we are not able to view third level menu when will come on second level category menu with fast mouce hover 

### Expected result (*)

1. When user will hover the second level category menu third lavel menu shuld be display..

### Actual result (*)

1. In short we are not able to view third level menu when will come on second level category menu with fast mouce hover 
http://prntscr.com/lsb7sw

**Comment by magento-engcom-team** [comment]
Hi @abrarpathan19. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [x] Summary of the issue
- [x] Information on your environment
- [x] Steps to reproduce
- [x] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me $VERSION instance` 

where `$VERSION` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (for example: 2.3-develop). 
For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@abrarpathan19 do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [x] yes
- [ ] no

**Comment by magento-engcom-team** [comment]
Hi @hardik-krish. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [ ] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [ ] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by hardik-krish** [comment]
I am able to reproduce this issue.

**Comment by hardik-krish** [comment]
Working on this issue.

**Comment by rogyar** [comment]
Hi @abrarpathan19. Thank you for the report. I'm not sure we are talking about an "issue" here since the described behavior is the description of how the menu exactly works. 
If we consider this as an issue, we may also say that if you quickly move the pointer over the top level menu items - the submenu items are not being shown and it's a bug. But it's not.
The delay there is set to prevent the situation when during moving the pointer across a page you have the main menu "shooting" at you.

**Comment by abrarpathan19** [comment]
@rogyar 
Thanks for the quick response, Please review below Bug video link  https://drive.google.com/file/d/1hHv90DoF60iZLmIEKLTN7p3hc0QpHtxU/view

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-nazar. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [x] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [x] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by ghost** [comment]
HI @abrarpathan19 thank you for you report, i'm not able to reproduce following steps you described, i dont think that this is the bug, please look at attachments.
![menu](https://user-images.githubusercontent.com/40756439/49721277-8dc48680-fc6a-11e8-8f08-ce2441a7a3f1.gif)
